torchvision/csrc/ROIAlign.h

```diff
@@ -4,6 +4,7 @@
 
 #ifdef WITH_CUDA
 #include "cuda/vision_cuda.h"
+#include "autocast.h"
```

Comment:
Tests are failing because this is only included on CUDA builds, but the `autocast` is used unconditionally.

Maybe you could add another `ifdef WITH_CUDA` around `ROIALIGN_autocast` ?

Reply:
Good catch, thanks!